openassessment/assessment/api/peer.py

```diff
@@ -565,7 +565,7 @@ def get_submitted_assessments(submission_uuid, scored_only=True, limit=None):
 
     Keyword Arguments:
         scored (boolean): Only retrieve the assessments used to generate a score
-            for this submission.
+            for their respective submission.
```

Comment:
This seemed to be incorrectly copied-but-never-updated documentation, let me know if this fix is incorrect.